docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md

```diff
@@ -247,6 +247,35 @@ upstart 流需要一个唯一的密钥，所以我们必须声明 `_id` 作为
         <td>Long</td>
         <td>起始毫秒数, 仅适用于 <code>'timestamp'</code> 启动模式.</td>
     </tr>
+    <tr>
+      <td>initial.snapshotting.queue.size</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">10240</td>
+      <td>Integer</td>
+      <td>进行初始快照时的队列大小。仅在 scan.startup.mode 选项设置为 initial 时生效。
+          注意：已弃用的选项名是 copy.existing.queue.size，为了兼容旧版本的作业，该选项名仍可用，但是推荐升级到新选项名
```

Comment:
nit: insert a `<br>` between description and deprecation notice.

Reply:
Resolved.